packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/base/data_sources/remote/users_remote_data_source.dart

```diff
@@ -0,0 +1,37 @@
+{{> licence.dart }}
+
+import 'package:dio/dio.dart';
+import 'package:retrofit/error_logger.dart';
+import 'package:retrofit/http.dart';
+
+import '../../models/credentials_model.dart';
+import '../../models/request_models/confirm_email_model.dart';
+import '../../models/user_model.dart';
+import '../../models/user_with_auth_token_model.dart';
+
+part 'users_remote_data_source.g.dart';
+
+@RestApi()
+abstract class UsersRemoteDataSource {
```

Comment:
Same comment as above

Reply:
Hmm do you think it will be better to have two data sources ?

Users
- GET /api/users/me
- PATCH /api/users/me

Register
- /api/register
- /api/register/me/email/resend-confirmation
- /api/register/me/email/confirm
- /api/register/me/phone/confirm
- /api/register/me/phone/resend

Reply:
I do think it's better to have them separate. I'll change it, but that means I should also update the API contract to use the new endpoints.

btw, if we're separating them, do you think it makes sense to remove `/me` from the register endpoints? Since the user is always registering themselves.
Like so:

Users

- GET /api/users/me
- PATCH /api/users/me

Register

- /api/register
- /api/register/email/resend-confirmation
- /api/register/email/confirm
- /api/register/phone/confirm
- /api/register/phone/resend

Reply:
I agree

Reply:
Done